// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript2018/asyncGenerators/parser.asyncGenerators.objectLiteralMethods.es2018.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,22 +1,16 @@
 
 
 //// [/.src/asyncGeneratorGetAccessorIsError.d.ts]
-declare const o22: {
-    get(): any;
-    x(): number;
-};
+declare const o22: invalid;
 
 //// [/.src/asyncGeneratorPropertyIsError.d.ts]
 declare const o24: {
     x(): 1;
 };
 
 //// [/.src/asyncGeneratorSetAccessorIsError.d.ts]
-declare const o23: {
-    set(): any;
-    x(value: number): void;
-};
+declare const o23: invalid;
 
 //// [/.src/awaitAsTypeIsOk.d.ts]
 interface await {
 }
@@ -126,10 +120,12 @@
     f(): AsyncGenerator<number, void, unknown>;
 };
 /// [Errors] ////
 
+asyncGeneratorGetAccessorIsError.ts(2,13): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 asyncGeneratorGetAccessorIsError.ts(2,17): error TS1005: '(' expected.
 asyncGeneratorPropertyIsError.ts(2,14): error TS1005: '(' expected.
+asyncGeneratorSetAccessorIsError.ts(2,13): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 asyncGeneratorSetAccessorIsError.ts(2,17): error TS1005: '(' expected.
 awaitInParameterInitializerIsError.ts(2,27): error TS2524: 'await' expressions cannot be used in a parameter initializer.
 awaitMissingValueIsError.ts(3,14): error TS1109: Expression expected.
 awaitParameterIsError.ts(2,15): error TS1359: Identifier expected. 'await' is a reserved word that cannot be used here.
@@ -293,19 +289,23 @@
                         ~~~~~~~
 !!! error TS2464: A computed property name must be of type 'string', 'number', 'symbol', or 'any'.
         }
     };
-==== asyncGeneratorGetAccessorIsError.ts (1 errors) ====
+==== asyncGeneratorGetAccessorIsError.ts (2 errors) ====
     const o22 = {
         async * get x(): number {
+                ~~~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                     ~
 !!! error TS1005: '(' expected.
             return 1;
         }
     };
-==== asyncGeneratorSetAccessorIsError.ts (1 errors) ====
+==== asyncGeneratorSetAccessorIsError.ts (2 errors) ====
     const o23 = {
         async * set x(value: number): void {
+                ~~~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                     ~
 !!! error TS1005: '(' expected.
         }
     };
